src/zope/app/form/browser/tests/test_form.py

```diff
@@ -21,19 +21,31 @@
 from zope.schema.interfaces import ITextLine
 from zope.component import testing
 
-from zope.app.testing import ztapi
+import zope.app.form.testing as ztapi
 
 from zope.app.form.browser import TextWidget
 from zope.formlib.interfaces import IInputWidget
 
+
+from zope.testing import renormalizing
+
+import re
+checker = renormalizing.RENormalizing([
+    (re.compile("u('.*?')"), r"\1"),
+    (re.compile('u(".*?")'), r"\1"),
```

Comment:
Ah, the regex is too greedy!  Replacing `.*` with `[^']*` would've probably worked (does Python have regex syntax for a non-greedy `*` a la Perl's `*?`?) , but the print solution is fine too.